generators/form/index.js

```diff
@@ -1,39 +1,57 @@
 'use strict';
 const Generator = require('yeoman-generator');
 
+/**
+ * Helper that returns a date one year from today in the dumb standard USA style
+ * of M/D/YYYY
+ * 8/3/2019
+ */
+function getDate() {
+  const now = new Date();
+  const month = now.getMonth();
+  const date = now.getDate();
+  const year = now.getFullYear() + 1;
+  return `${month}/${date}/${year}`;
+}
+
 module.exports = class extends Generator {
   prompting() {
     const prompts = [
       {
         type: 'input',
         name: 'formNumber',
-        message: "What's your form number?"
+        message: "What's your form number?",
+        default: 'XX-230'
       },
       {
         type: 'input',
         name: 'trackingPrefix',
-        message: "What's the Google Analytics event prefix you want to use?"
+        message: "What's the Google Analytics event prefix you want to use?",
+        default: `${this.options.folderName}-`
       },
       {
         type: 'input',
         name: 'respondentBurden',
-        message: "What's the respondent burden of this form in minutes?"
+        message: "What's the respondent burden of this form in minutes?",
+        default: '30'
       },
       {
         type: 'input',
         name: 'ombNumber',
-        message: "What's the OMB control number for this form?"
+        message: "What's the OMB control number for this form?",
+        default: answers => answers.formNumber
       },
       {
         type: 'input',
         name: 'expirationDate',
-        message: "What's the OMB expiration date for this form?"
+        message: "What's the OMB expiration date for this form?",
+        default: getDate
       },
       {
         type: 'input',
         name: 'benefitDescription',
-        message:
-          "What's the benefit description for this form? Something like 'health care benefits'"
+        message: "What's the benefit description for this form?",
+        default: `${this.options.appName} Benefits`
```

Comment:
Related to the comment on form title above, depending on how appName is used, this could yield some redundant language.

We currently refer to these internally (i.e. variable name) as benefits, but since it looks like the benefit values don't always include the word "benefit", perhaps it would be helpful to provide a parenthetical example, and ask instead what "type" of form or application is this (e.g. opt out, dependent status)?

```
  export const formBenefits = {
  '21-526EZ': 'increased disability compensation',
  '21P-527EZ': 'Veterans pension benefits',
  '21P-530': 'burial benefits',
  '1010ez': 'health care',
  '22-0993': 'opt out',
  '22-1990': 'education benefits',
  '22-1990E': 'education benefits',
  '22-1990N': 'education benefits',
  '22-1995': 'education benefits',
  '22-5490': 'education benefits',
  '22-5495': 'education benefits',
  '40-10007': 'pre-need determination of eligibility in a VA national cemetery',
  VIC: 'Veteran ID Card',
  '21-686C': 'dependent status'
};
```

Reply:
And in that case we could even prompt them with multiple choice to pick from. I think... I've seen that kind of snazziness in a command line tool but don't know if it was Yeoman.